test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,130 @@
+    @alexander = Contestant.new({
+                       first_name: 'Alexander',
+                       last_name: 'Aigades',
+                       age: 28,
+                       state_of_residence: 'CO',
+                       spending_money: 10})
+    @benjamin = Contestant.new({
+                       first_name: 'Benjamin',
+                       last_name: 'Franklin',
+                       age: 17,
+                       state_of_residence: 'PA',
+                       spending_money: 100})
+    @frederick = Contestant.new({
+                       first_name:  'Frederick',
+                       last_name: 'Douglas',
+                       age: 55,
+                       state_of_residence: 'NY',
+                       spending_money: 20})
+    @winston = Contestant.new({
+                       first_name: 'Winston',
+                       last_name: 'Churchill',
+                       age: 18,
+                       state_of_residence: 'CO',
+                       spending_money: 5})
+    @grace = Contestant.new({
+                       first_name: 'Grace',
+                       last_name: 'Hopper',
+                       age: 20,
+                       state_of_residence: 'CO',
+                       spending_money: 20})
+
+    @alexander.add_game_interest('Pick 4')
+    @alexander.add_game_interest('Mega Millions')
+    @frederick.add_game_interest('Mega Millions')
+    @winston.add_game_interest('Cash 5')
+    @winston.add_game_interest('Mega Millions')
+    @benjamin.add_game_interest('Mega Millions')
+    @grace.add_game_interest('Mega Millions')
+    @grace.add_game_interest('Cash 5')
+    @grace.add_game_interest('Pick 4')
+  end
+
+  def test_existence
+    assert_instance_of ColoradoLottery, @lottery
+  end
+
+  def test_attributes
+    assert_equal({}, @lottery.registered_contestants)
+    assert_equal [], @lottery.winners
+    assert_equal({}, @lottery.current_contestants)
+  end
+
+  def test_interest_and_age
+    assert_equal true, @lottery.interested_and_18?(@alexander, @pick_4)
+    assert_equal false, @lottery.interested_and_18?(@benjamin, @mega_millions)
+    assert_equal false, @lottery.interested_and_18?(@alexander, @cash_5)
+  end
+
+  def test_can_register
+    assert_equal true, @lottery.can_register?(@alexander, @pick_4)
+    assert_equal false, @lottery.can_register?(@alexander, @cash_5)
+    assert_equal true, @lottery.can_register?(@frederick, @mega_millions)
+    assert_equal false, @lottery.can_register?(@benjamin, @mega_millions)
+    assert_equal false, @lottery.can_register?(@frederick, @cash_5)
+  end
+
+  def test_contestant_registration
+    @lottery.register_contestant(@alexander, @pick_4)
+    assert_equal [@alexander], @lottery.registered_contestants["Pick 4"]
+    @lottery.register_contestant(@alexander, @mega_millions)
+    assert_equal [@alexander], @lottery.registered_contestants["Mega Millions"]
+    assert_equal 2, @lottery.registered_contestants.keys.length
```

Comment:
with very little exception, avoid using length as a test assertion--it's essentially a meaningless test.  just assert the exact keys you want. better yet, assert the hash exactly as you'd like it to look to make sure that you're getting exactly what you want. that will also be helpful to future developers trying to figure out what your code should be returning.